### DISCUSSION TITLE
method to download db2 on windows

### DISCUSSION THREAD

**Discussion by masterflinter** [aquasecurity/trivy#4707] [Q&A]
### Question

Is there a method to download db2 for scanning in an airgapped environment on windows? My situation is that I must use a windows machine to download the offline db then transfer it into an airgapped linux environment. 

Actually this no longer works in v42. --download-db-only is not supported. Is airgapped not supported going forward? Also, I never seem to be able to get a db newer than the final v1 released in feb 2023.

Is there no currently functioning method to update the trivy db in an airgapped environment?

### Target

Container Image

### Scanner

Vulnerability

### Output Format

JSON

### Mode

Standalone

### Operating System

windows

**Comment by knqyf263** [comment]
> Actually this no longer works in v42. --download-db-only is not supported

What do you mean?

```
$ trivy image -h | grep 'download-db-only' 
      --download-db-only            download/update vulnerability database but don't run a scan
```
